.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
```

Comment:
Let us add `"parser": "babel-eslint"` in line 7

Reply:
After adding that line, I got errors: Failed to load parser 'babel-eslint' declared in '.eslintrc.json': Cannot find module 'babel-eslint'